pkgs/by-name/un/unfloader/package.nix

```diff
@@ -0,0 +1,55 @@
+{lib, stdenv, fetchFromGitHub, ncurses, libftdi1, libusb1, pkg-config}:
+
+stdenv.mkDerivation rec {
+    pname = "UNFLoader";
+    version = "2.2";
+
+    src = fetchFromGitHub {
+        owner = "buu342";
+        repo = "N64-UNFLoader";
+        rev = "v${version}";
+        hash = "sha256-nps1xhp/gLJspMr1cua7wvD9PoXte/fB8XJB7559D+k=";
+    };
+
+    nativeBuildInputs = [
+        pkg-config
+    ];
+
+    buildInputs = [
+        ncurses
+        libftdi1
+        libusb1
+    ];
+
+    # Patch the Makefile to use dynamic libraries instead of static ones
+    preBuild = ''
+        export NIX_LDFLAGS="-lusb-1.0 -lftdi1 $NIX_LDFLAGS"
+
+        # Create symlinks with the static library names pointing to the shared libraries
+        mkdir -p staticlibs
+        ln -sf ${libusb1}/lib/libusb-1.0.so staticlibs/libusb-1.0.a
+        ln -sf ${libftdi1}/lib/libftdi1.so staticlibs/libftdi1.a
+        export LIBRARY_PATH=$LIBRARY_PATH:$(pwd)/staticlibs
+    '';
+    sourceRoot = "${src.name}/UNFLoader";
+
+    installPhase = ''
+        mkdir -p $out/bin
+        cp ${pname} $out/bin/${pname}
+        chmod +x $out/bin/${pname}
+    '';
+
+    meta = with lib; {
```

Comment:
Let's move out the `with lib;` to single expression to improve maintainability:
- #208242